File: Data Structures & Algorithms/duplicate-integer/submission-1.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums: number[]): boolean {
        for (let i: number = 0; i < nums.length; i++) {
            for (let j: number = i+1; j < nums.length; j++) {
                if (nums[i] === nums[j]) {
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums.sort()
        print(nums)
        number = 0
        countHolder = 0;
        numHolder = 0;

        if len(nums) == 0:
            return 0

        for i in range(len(nums)):
            if i == 0:
                countHolder+=1
                numHolder = nums[i]
            else:
                if nums[i] == nums[i-1]:
                    continue
                elif nums[i] - 1 == numHolder:
                    countHolder+=1
                    numHolder = nums[i]
                else:
                    countHolder = 1
                    numHolder = nums[i]
            if (countHolder > number):
                number = countHolder
        return number